### PR TITLE
[NUI] Add KeyInputFocus property to View

### PR DESCRIPTION
**Pull request by Seoyeon2Kim**
Signed-off-by: Seoyeon Kim <seoyeon2.kim@samsung.com>

### Description of Change ###
- KeyInputFocus is to receive key events to View.
- This property is originally public on DALi package,
 so to use it in NUI app, it should be supplied even as hidden API.

### API Changes ###
- N/A

### CODE REVIEW

**Review by JoogabYun — APPROVED** [review]

File: src/Tizen.NUI/src/public/BaseComponents/View.cs [hunk 1]
```diff
@@ -243,6 +243,23 @@ public string StyleName
             }
         }
 
+        /// <summary>
+        /// The KeyInputFocus, type bool.
```
Comment (dasg34) [inline src/Tizen.NUI/src/public/BaseComponents/View.cs:247]: hmm.. `KeyInputFoucs` type is string...

File: src/Tizen.NUI/src/public/BaseComponents/ViewBindableProperty.cs [hunk 2]
```diff
@@ -46,6 +46,26 @@ public partial class View
...
+            var view = (View)bindable;
+            if (newValue != null)
+            {
+                Tizen.NUI.Object.SetProperty(view.swigCPtr, View.Property.KEY_INPUT_FOCUS, new Tizen.NUI.PropertyValue((bool)newValue));
+            }
+        },
+        defaultValueCreator: (bindable) =>
+        {
+            var view = (View)bindable;
+            string temp;
```
Comment (dasg34) [inline src/Tizen.NUI/src/public/BaseComponents/ViewBindableProperty.cs:64]: It may be `bool`.

**Review by dasg34 — CHANGES_REQUESTED** [review]

File: src/Tizen.NUI/src/public/BaseComponents/ViewBindableProperty.cs (hunk 2 above)
Comment (Seoyeon2Kim) [reply src/Tizen.NUI/src/public/BaseComponents/ViewBindableProperty.cs]: Oh You're right.
I did the wrong type here. Fixed it.
Thank you for reviewing it!!

File: src/Tizen.NUI/src/public/BaseComponents/View.cs (hunk 1 above)
Comment (Seoyeon2Kim) [reply src/Tizen.NUI/src/public/BaseComponents/View.cs:247]: Changed to 'bool' type of `KeyInputFocus`

**Review by dasg34 — APPROVED** [review]

**Review by AdunFang — APPROVED** [review]
It is OK.

**Pull request merged**